Bottom: b0bc45790554f16daab42de192411a34d11563e1
Top:    b35e3ea5b3ce5ff1edd365ef593922a1334d9adb
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-10 14:27:30 -0500

Add variable file for HSM host configuration


---
diff --git a/ansible-playbook/group_vars/buildbot/hsm.yml b/ansible-playbook/group_vars/buildbot/hsm.yml
new file mode 100644
index 00000000..aa3e1150
--- /dev/null
+++ b/ansible-playbook/group_vars/buildbot/hsm.yml
@@ -0,0 +1,2 @@
+---
+hsm_host: "172.16.64.38"
